README.md

```diff
@@ -43,7 +43,7 @@ Mitaka is a browser extension for OSINT search which can:
 ### Supported search engines
 
 | name                 | url                                    | supported types                   |
-|:---------------------|:---------------------------------------|:----------------------------------|
```

Comment:
Why do you change the column headers format?

Reply:
I was using an Atom editor extension that automatically changed it - it doesn't affect the markdown rendering 🤷‍♂️